doc/api_blueprints/elektrad.apib

```diff
@@ -15,120 +15,156 @@ The API version is increased whenever breaking changes
 The Elektra version is directly taken from the Elektra library,
 for further information and explanation see [doc/VERSION.md](https://github.com/ElektraInitiative/libelektra/blob/master/doc/VERSION.md).
 
-+ Response 200 (application/json)
++ Response 200 (application/json; charset=utf-8)
     + Attributes (object)
+        + api: 1 (number) - The version of the API itself
         + elektra (object) - Detailed version information about the used Elektra library
-            + version (string) - The currently used version in its complete format
-            + major (number) - The currently used major version
-            + minor (number) - The currently used minor version
-            + micro (number) - The currently used micro version
-        + api (number) - The version of the API itself
+            + version: 0.8.23 (string) - The currently used version in its complete format
+            + major: 0 (number) - The currently used major version
+            + minor: 8 (number) - The currently used minor version
+            + micro: 23 (number) - The currently used micro version
 
 
-## elektra key database [/kdb/{+path}{?apikey}]
+## elektra key database [/kdb/{+path}]
 
 access the elektra key database by specifying a `path`
 
-+ Parameters
-    + path: `user/hello` (string) - path to the elektra config
-    + apikey: `7b7947e6-adc5-4d23-b53c-783623f894f8` (string, required) - API key used to authenticate with `elektrad`
-
 ### get configuration [GET]
 
-+ Response 200 (application/json)
+this actually does `kdb get`, `kdb ls` and `kdb lsmeta`/`kdb getmeta` at once and is used to browse the kdb
```

Comment:
why "actually"?, the rest of the sentence is also a rather poor description. ("at once"? "browse"?)